content/_help/verify-your-identity/verify-your-identity-in-person._fr.md

```diff
@@ -0,0 +1,103 @@
+
+## Commencez cette procédure sur Login.gov avant de vous rendre au bureau de poste
+
+Veuillez préparer les informations suivantes pour les saisir sur Login.gov:
+
+1. Votre carte d'identité délivrée par l'État. [Vous n'avez pas de carte d'identité délivrée par l'État?](/fr/help/verify-your-identity/accepted-state-issued-identification/)
+2. Votre numéro de sécurité sociale
+3. Un numéro de téléphone sur un forfait téléphonique qui est à votre nom
+   * Si vous n'avez pas de forfait téléphonique à votre nom, nous pouvons vous envoyer le code de vérification par courrier, ce qui prend environ trois à cinq jours.
+
+Si vous ne parvenez pas à effectuer l'une de ces étapes, veuillez contacter le centre d'assistance de l'agence partenaire pour obtenir de l'aide.
+
+## Apportez les articles suivants au bureau de poste
+
+### Une copie de votre code-barres envoyé par courriel
+
+Une fois que vous aurez complété les étapes sur Login.gov, nous vous enverrons par courriel un code-barres et une date limite pour vous rendre au bureau de poste. Vous pouvez imprimer le courriel, ou le montrer depuis votre cellulaire. Si vous vous rendez au bureau de poste après la date limite, vos informations ne seront pas sauvegardées et vous devrez recommencer le processus.
+
+### Votre carte d'identité délivrée par l'État
+
+Apportez la même carte d'identité que vous avez utilisée sur Login.gov. Votre pièce d'identité ne doit pas être périmée. Pour l'instant, seules les pièces d'identité suivantes, délivrées par l'État, sont acceptées:
+
+* Un permis de conduire des 50 États, du district de Columbia (D. C.) et des autres territoires américains (Guam, îles Vierges américaines, îles Mariannes et Porto Rico).
+* Une carte d'identité sans permis de conduire délivrée par l'État.
+  * Il s'agit d'un document d'identité délivré par l'État, le district de Columbia (D. C.) ou le territoire américain, qui atteste de l'identité mais ne donne pas le droit de conduire.
+
+Nous n'acceptons actuellement aucune autre forme d'identification, comme les passeports et les cartes d'identité militaires. Vous ne pouvez pas vérifier votre identité au bureau de poste sans un document d'identité délivré par l'État. Nous travaillons actuellement à l'ajout d'autres moyens de vérifier votre identité.
+
+En attendant, veuillez contacter le centre d'assistance de l'agence partenaire pour savoir ce que vous pouvez faire.
+
+### Preuve de votre adresse actuelle
+
+Si votre adresse actuelle ne correspond pas à l'adresse figurant sur votre pièce d'identité, vous devrez apporter une preuve de votre adresse actuelle au bureau de poste. Les formes acceptables de preuve d'adresse sont:
+
+* Bail, hypothèque ou acte de fiducie
+* Inscription sur les listes électorales
+* Carte d'immatriculation d’un véhicule
+* Police d'assurance habitation ou véhicule
+
+Vous pouvez apporter une copie physique ou montrer une copie numérique du document. Vous ne pouvez pas montrer une photo ou une capture d'écran du document.
```

Comment:
Preparing suggestion in case we need to swap it out:

```suggestion
Vous devez apporter une copie physique du document. Vous ne pouvez pas utiliser une photo ou une capture d'écran du document.
```